crates/composefs-boot/src/write_boot.rs

```diff
@@ -71,7 +71,6 @@ pub fn write_t2_simple<ObjectID: FsVerityHashValue>(
 /// * entry          - Boot entry variant to be written
 /// * root_id        - The content hash of the generated EROFS image id
 /// * boot_partition - Path to the boot partition/directory
-/// * entry_id       - In case of a BLS entry, the name of file to be generated in `loader/entries`
 /// * boot_subdir    - If `Some(path)`, the path is prepended to `initrd` and `linux` keys in the BLS entry
 ///
 /// For example, if `boot_subdir = Some("/boot/1")` and `boot_partition = "/boot"`,
```

Comment:
The repetition of `/boot` is the thing that I'm objecting to.  The path to the files contained in the entry should consist of concatenating boot_partition + boot_subdir + entry_id.  I think we should talk about this in a call :)